docs/revsets.md

```diff
@@ -92,19 +92,17 @@ revsets (expressions) as arguments.
   pattern](#string-patterns). For example, `branches(push)` would match the
   branches `push-123` and `repushed` but not the branch `main`. If a branch is
   in a conflicted state, all its possible targets are included.
-
-*   `remote_branches([branch_pattern[, [remote=]remote_pattern]])`: All remote
-    branch targets across all remotes. If just the `branch_pattern` is
-    specified, the branches whose names match the given [string
-    pattern](#string-patterns) across all remotes are selected. If both
-    `branch_pattern` and `remote_pattern` are specified, the selection is
-    further restricted to just the remotes whose names match `remote_pattern`.
-    
-    For example, `remote_branches(push, ri)` would match the branches
-    `push-123@origin` and `repushed@private` but not `push-123@upstream` or
-    `main@origin` or `main@upstream`. If a branch is in a conflicted state, all
-    its possible targets are included.
-
+* `remote_branches([branch_pattern[, [remote=]remote_pattern]])`: All remote
+  branch targets across all remotes. If just the `branch_pattern` is
+  specified, the branches whose names match the given [string
+  pattern](#string-patterns) across all remotes are selected. If both
+  `branch_pattern` and `remote_pattern` are specified, the selection is
+  further restricted to just the remotes whose names match `remote_pattern`.
+
+  For example, `remote_branches(push, ri)` would match the branches
+  `push-123@origin` and `repushed@private` but not `push-123@upstream` or
+  `main@origin` or `main@upstream`. If a branch is in a conflicted state, all
+  its possible targets are included.
```

Comment:
nit: I slightly prefer having empty lines between list items (so that multi-paragraph list item wouldn't look split towards neighbor items.)